service/model/interface.go

```diff
@@ -58,3 +58,11 @@ type ProjectInvitationRepository interface {
 type ProjectService interface {
 	Get(ctx context.Context, projectID, authUserID uuid.UUID) (Project, error)
 }
+
+type ProjectInvitationSender interface {
+	SendProjectInvitation(ctx context.Context, input ProjectInvitationInput)
+}
```

Comment:
Interface for sending project invitations. The service now does not need to know whether the invitation is sent via email or another channel